tests/cpp_api/audio_tests/audio_tests.cpp

```diff
@@ -217,22 +304,32 @@ int test(int test_case, const char *path, int qa_mode, int downmix, int gpu) {
         int *label_id = reinterpret_cast<int *>(labels->at(0)->buffer());  // The labels are present contiguously in memory
         std::cout << "Audio file : " << audio_file_name << "\n";
         std::cout << "Label : " << *label_id << "\n";
-        for (uint idx = 0; idx < output_tensor_list->size(); idx++) {
-            buffer = static_cast<float *>(output_tensor_list->at(idx)->buffer());
-            output_tensor_list->at(idx)->copy_roi(roi.data());
-            max_channels = output_tensor_list->at(idx)->dims().at(2);
-            max_samples = max_channels == 1 ? 1 : output_tensor_list->at(idx)->dims().at(1);
-            frames = roi[idx * 4 + 2];
-            channels = roi[idx * 4 + 3];
-            buffer_size = roi[idx * 4 + 2] * roi[idx * 4 + 3];
+        if(test_case == 8){
```

Comment:
please mention a comment stating what test case 8 is.

Reply:
Added a comment for test case 8